packages/backend/src/models/OpenIdIdentitiesModel.ts

```diff
@@ -144,4 +148,15 @@ export class OpenIdIdentityModel {
                 .delete();
         });
     }
+
+    async getRefreshToken(userId: number) {
```

Comment:
We almost always use `userUuid` in the model layers

Reply:
I guess this can return `null` although it's not in the return type. 

Should this be `findRefreshToken`? Usually we throw an error in methods named `get..` instead of returning undefined.